src/debug-generator/debug-generator.c

```diff
@@ -142,6 +159,21 @@ static int generate_wants_symlinks(void) {
         return r;
 }
 
+static int install_debug_shell_dropin(const char *dir, const char *tty) {
+        int r;
+
+        r = write_drop_in_format(dir, "debug-shell.service", 50, "tty",
+                        "[Unit]\n"
+                        "Description=Early root shell on /dev/%s FOR DEBUGGING ONLY\n"
```

Comment:
Please add `ConditionPathExists=` here, (i.e. with an empty string) to reset the condition list, but leave it in the normal file.